assets/js/modules/analytics/datastore/accounts.test.js

```diff
@@ -88,7 +89,7 @@ describe( 'modules/analytics accounts', () => {
 					{ accountName, propertyName, profileName, timezone }
 				);
 
-				expect( registry.select( STORE_NAME ).getAccountTicketTermsOfServiceURL() ).toEqual( `https://analytics.google.com/analytics/web/?provisioningSignup=false#management/TermsOfService/?api.accountTicketId=${ fixtures.createAccount.id }` );
+				expect( store.getState().accountTicketID ).toEqual( fixtures.createAccount.id );
```

Comment:
Normally I think we shouldn't reach into the state directly, but in this case I think it's okay.